doc/source/whatsnew/v0.18.1.rst

```diff
@@ -53,6 +53,7 @@ Friday before MLK Day
 
 .. ipython:: python
 
+    from datetime import datetime
```

Comment:
can you use `import datetime` instead, and use `datetime.datetime(...)` below

Reply:
Sure. May I ask why you prefer that?
Is that because it can cause some namespace issues?

Reply:
There are many reasons, and some are not trivial to explain.

The main one is that if I see a bare `datetime` in Python code, I need to guess whether it's the main module, or the `datetime.datetime` module. While there is no ambiguity in reading `datetime.datetime`.

Then, with a `from datetime import datetime` I load the whole `datetime` module in `sys.modules`, but I only have access in the namespace to the `datetime.datetime` submodule (with the name `datetime`). This won't let me perform operations like `imp.reload(datetime)`, as I don't have anything in the namespace pointing to the main `datetime` module.